docs/changehistory/NextVersion.md

```diff
@@ -71,3 +74,9 @@ A new optional [`extendedData`]($docs/presentation/content/CalculatedPropertiesS
 - `LayoutFragmentProps`, `ContentLayoutProps`, `LayoutSplitPropsBase`, `LayoutHorizontalSplitProps`, `LayoutVerticalSplitProps`, and `StandardContentLayouts` have been deprecated. Use the same APIs from `@itwin/appui-react` instead.
 
 - `BackendItemsManager` is internal and should never have been consumed. It has been deprecated and will be removed in 5.0.0. Use `UiFramework.backstage` from `@itwin/appui-react` instead.
+
+## Deprecations
+
+### @itwin/core-frontend
+
+- [SnapshotConnection.openRemote]($frontend) is deprecated. To open iModel for web workflows use [CheckpointConnection.openRemote]($frontend) instead.
```

Comment:
This should be merged with "API deprecations section". Table of contents needs to be fixed as well.